sos/report/__init__.py

```diff
@@ -1397,11 +1404,19 @@ def ui_progress(self, status_line):
     def collect_env_vars(self):
         if not self.env_vars:
             return
-        env = '\n'.join([
-            "%s=%s" % (name, val) for (name, val) in
-            [(name, '%s' % os.environ.get(name)) for name in self.env_vars if
-             os.environ.get(name) is not None]
-        ]) + '\n'
+        env = (
+            '\n'.join(
+                [
+                    f"{name}={val}"
+                    for (name, val) in [
+                        (name, f'{os.environ.get(name)}')
+                        for name in self.env_vars
+                        if os.environ.get(name) is not None
+                    ]
+                ]
+            )
+            + '\n'
+        )
```

Comment:
Conversely to the other nesting changes, this one got a bit blown out, I think we can button this up a bit most likely.